src/API/WesternStatesWater.WestDaat.Tests.ManagerTests/SystemManagerTests.cs

```diff
@@ -11,6 +11,8 @@ public class SystemManagerTests : ManagerTestBase
         private readonly Mock<ILocationEngine> _locationEngineMock = new(MockBehavior.Strict);
 
         private readonly Mock<ISystemAccessor> _systemAccessorMock = new(MockBehavior.Strict);
+        
+        [TestMethod]
```

Comment:
? lol

Reply:
lol yeah, my bad. Future commit will be actually using that 😄